app/reducers/loginError.jsx

```diff
@@ -0,0 +1,22 @@
+
+const SET_ERROR = 'SET_ERROR'
+export const setLoginError = errorState => ({
+  type: SET_ERROR,
+  errorState  // boolean
+})
+
+export const clearLoginError = () => ({
```

Comment:
I debated those two options. I thought clearLoginError was just a little clearer though. I guess really it is duplicate code though. I can refactor that real quick